sherlock/sherlock.py

```diff
@@ -474,14 +474,35 @@ def timeout_check(value):
     return timeout
 
 
-def main():
+def check_sherlock_version():
+    """
+     Check for newer version of Sherlock. If it exists, let the user know
+     about it.
+    """
+    try:
+        r = requests.get(
+            "https://raw.githubusercontent.com/sherlock-project/sherlock/master/sherlock/sherlock.py")
+
+        remote_version = str(re.findall('__version__ = "(.*)"', r.text)[0])
+        local_version = __version__
+
+        if remote_version != local_version:
+            print("Update Available!\n" +
+                  f"You are running version {local_version}. Version {remote_version} is available at https://git.io/sherlock")
+
+    except Exception as error:
+        print(f"A problem occurred while checking for an update: {error}")
+
+
+def adding_arguments(parser):
```

Comment:
The name of this function is not really clear about what it does. The verb shouldn't be in infinitive, **_add_arguments_** would be a better naming convention